packages/tupaia-web/src/features/Map/MapOverlays/DataVisualsLayer.tsx

```diff
@@ -24,26 +27,44 @@ const ShadedPolygon = styled(Polygon)`
   }
 `;
 
+const useNavigateToEntity = () => {
```

Comment:
There is a util we already have called `useEntityLink` you might be able to use instead of this